src/sa_web/static/js/views/app-view.js

```diff
@@ -220,7 +224,7 @@ var Shareabouts = Shareabouts || {};
         S.Config.flavor.app.list_enabled) {
           this.listView = new S.PlaceListView({
             el: '#list-container',
-            collection: this.collection
+            collection: this.collection.places
```

Comment:
I'm sure this is on your todo list, but we should definitely abstract this to either iterate through all places instead of hard-coding the `places` key.